autoload/youcompleteme.vim

```diff
@@ -84,6 +84,9 @@ function! youcompleteme#Enable()
     autocmd InsertLeave * call s:OnInsertLeave()
     autocmd InsertEnter * call s:OnInsertEnter()
     autocmd VimLeave * call s:OnVimLeave()
+    if pyeval( 'vimsupport.VimVersionAtLeast("7.3.598")' )
```

Comment:
I appreciate this, but just bump the global required Vim version. It's pretty close to the version we require already.